pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -317,29 +316,11 @@ private String getDynamicConfigurationFromStore(String path, String settingName,
     }
 
     private double getDynamicConfigurationDouble(String path, String settingName, double defaultValue) {
-        double result = defaultValue;
-        try {
-            String setting = this.getDynamicConfigurationFromStore(path, settingName, null);
-            if (setting != null) {
-                result = Double.parseDouble(setting);
-            }
-        } catch (Exception e) {
-            log.warn("Got exception when parsing configuration from path [{}]:", path, e);
-        }
-        return result;
+        return Double.parseDouble(getDynamicConfigurationFromStore(path, settingName, String.valueOf(defaultValue)));
```

Comment:
with this new code if there is something wrong written on the store we will throw a NumberFormatException
we should keep the "catch" statement

Reply:
yes，I will rollback.